tests/integration-tests/tests/scheduler_plugin/test_scheduler_plugin.py

```diff
@@ -620,3 +622,12 @@ def _test_compute_node_bootstrap_timeout(compute_node, compute_node_bootstrap_ti
     assert_that(str(base64.b64decode(user_data))).contains(
         f"timeout {compute_node_bootstrap_timeout} /tmp/bootstrap.sh"
     )
+
+
+def _test_log_file_permission(command_executor):
+    result = command_executor.run_remote_command("ls -l /var/log/parallelcluster/clusterstatusmgtd").stdout
```

Comment:
please consider using
```
$ stat -c '%a %U %G' /var/log/parallelcluster/clusterstatusmgtd
644 root root
```
then you can match with `equals`

Reply:
Good idea! Changed